docs/books/admin_guide/14-special-authority.md

```diff
@@ -0,0 +1,691 @@
+
+Look at the following example:
+
+```bash
+Shell > setfacl -m m:rwx /project
+Shell > setfacl -m u:tom:rx /project
+
+Shell > cd /project
+Shell > touch file1 file2
+# Because there is no recursion, the file here does not have ACL permission. 
+Shell > ls -l
+-rw-r--r-- 1 root root 0 Jan  12 14:35 file1
+-rw-r--r-- 1 root root 0 Jan  12 14:35 file2
+
+Shell > setfacl -m u:tom:rx -R /project
+Shell > ls -l /project
+-rw-r-xr--+ 1 root root 0 Jan  12 14:35 file1
+-rw-r-xr--+ 1 root root 0 Jan  12 14:35 file2
+```
+
+Now there is a question: if I create a new file in this directory, does it have ACL permission?
+The answer is no, because the newly created file is after the command `setfacl-m u:tom:rx -R / project` is executed.
+
+```bash
+Shell > touch /project/file3
+Shell > ls -l /project/file3
+-rw-r--r-- 1 root root 0 Jan  12 14:52 /project/file3
+```
+
+If you want the newly created directory/file to also have ACL permissions, you need to use default ACL permissions.
+
+```bash
+Shell > setfacl -m d:u:tom:rx  /project
+Shell > cd /project && touch file4 && ls -l 
+-rw-r-xr--+ 1 root root 0 Jan  12 14:35 file1
+-rw-r-xr--+ 1 root root 0 Jan  12 14:35 file2
+-rw-r--r--  1 root root 0 Jan  12 14:52 file3
+-rw-rw----+ 1 root root 0 Jan  12 14:59 file4
+
+Shell > getfacl -p /project
+# file: /project
+# owner: root
+# group: class1
+user::rwx
+user:tom:r-x
+group::rwx
+mask::rwx
+other::---
+default:user::rwx
+default:user:tom:r-x
+default:group::rwx
+default:mask::rwx
+default:other::---
+```
+
+### SetUID
+
+The role of "SetUID":
+
+* Only executable binaries can set SUID permissions.
+* The executor of the command should have x permission to the program.
+* The executor of the command obtains the identity of the owner of the program file when executing the program. 
+* The identity change is only valid during execution, and once the binary program is finished, the executor's identity is restored to the original identity.
+
+Why does GNU/Linux need such strange permissions?
+Take the most common `passwd` command as an example:
+
+![SetUID1](./images/SetUID1.png)
+
+As you can see, the ordinary users only has r and x, but the owner's x becomes s, proving that the `passwd` command has SUID permissions.
+
+It is well known that the ordinary users (uid >= 1000) can change his own password. The real password is stored in the **/etc/shadow** file, but the permission of the shadows file is 000, and the ordinary users does not have any permissions.
+
+```bash
+Shell > ls -l /etc/shadow
+---------- 1 root root 874 Jan  12 13:42 /etc/shadow
+```
+
+Since the ordinary users can change their password, they must have written the password to the **/etc/shadow** file. When an ordinary users executes the `passwd` command, it will temporarily change to the owner of the file -- **root**. For **shadow** file, **root** can not be restricted by permissions. This is why `passwd` command needs SUID permission.
```

Comment:
```suggestion
Since the ordinary users can change their password, they must have written the password to the **/etc/shadow** file. When an ordinary user executes the `passwd` command, it will temporarily change to the owner of the file -- **root**. For **shadow** file, **root** can not be restricted by permissions. This is why `passwd` command needs SUID permission.
```

Reply:
When an ordinary user...
(with the article an)
or
When ordinary users...
(without article)